src/Umbraco.Forms.Integrations.Crm.Hubspot/Extensions/RecordFieldExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Umbraco.Forms.Core.Data.Helpers;
+using Umbraco.Forms.Core.Persistence.Dtos;
+
+namespace Umbraco.Forms.Integrations.Crm.Hubspot.Extensions;
+
+public static class RecordFieldExtensions
+{
+    public static string ValuesAsHubspotString(this RecordField recordField, bool escaped = true)
+    {
+        if (!recordField.HasValue())
+        {
+            return string.Empty;
+        }
+
+        if (!escaped)
+        {
+            return string.Join(", ", recordField.Values.ToArray());
```

Comment:
This still looks like a "," to me?

Reply:
Indeed, good call. Thank you for observing this.
Updated